README.md

```diff
@@ -76,6 +76,29 @@ And this is what gets rendered (plus the CSS and Javascript you've specified):
 
 ## Release notes
 
+🚨📢 **Version 0.110**
+- BREAKING CHANGE:
+    - Installation changes:
+        - If your components include JS or CSS, you now must use the middleware and add django-components' URLs to your `urlpatterns`
+            (See "[Adding support for JS and CSS](#adding-support-for-js-and-css)")
+    - `preload` kwarg of `{% component_dependencies %}` tag was removed. This will be replaced with HTML fragment support.
+    - Component typing signature changed from `Component[Args, Kwargs, Data, Slots]` to `Component[Args, Kwargs, Slots, Data, JsData, CssData]`
+    - If you rendered a component A with `.render()` and then inserted that into another component B, now you must pass `kind=None` to component A:
+      ```py
+      prerendered_a = CompA.render(
+          args=[...],
+          kwargs={...},
+          kind=None,
+      )
+      html = CompB.render(
+          kwargs={
+              content=prerendered_a,
+          },
+      )
+      ```
+- `{% component_dependencies %}` tags are now OPTIONAL - If your components use JS and CSS, but you don't use `{% component_dependencies %}` tags, the JS and CSS will now be, by default, inserted at the end of `<body>` and at the end of `<head>` respectively.
```

Comment:
This is one change that was not discussed in the tech doc - I realized that we could make the use of `{% component_dependencies %}` tags optional. And, by default if no `{% component_dependencies %}` tags were inserted into the HTML document, we'd try to insert the CSS into the end of `<head>`, and JS at the end of `<body>`.

---

WIth this, hand-in-hand goes another question / consideration. IMO we should remove `{% component_dependencies %}` and instead keep only `{% component_js_dependencies %}` and `{% component_css_dependencies %}`.

Because having a single template tag that renders both JS and CSS is more convenient, so it's attractive to be used.

But from the perspective of web development best practices, placing the CSS and JS in the same places is NOT a good idea.

- Because either you put both JS and CSS into `<head>`, which means that execution of JS scripts will BLOCK the rendering of the rest of the page, potentially leading to a flash of empty page.
- And if both JS and CSS were placed at the end of `<body>`, then there may be a [flash of unstyled content](https://en.wikipedia.org/wiki/Flash_of_unstyled_content), because the HTML will be first rendered without any styling, and then re-rendered with the styling once the parser reaches the CSS styles at the end of the `<body>`.

So, instead, if we keep only `{% component_js_dependencies %}` and `{% component_css_dependencies %}`, users are forced to split the two up. And with a bit of luck, they google the best practices and insert the CSS / JS as we do so by default.